lib/src/main/java/org/exoplatform/social/addons/updater/RDBMSMigrationManager.java

```diff
@@ -140,6 +156,10 @@ public void run() {
                       && !MigrationContext.isIdentityDone()) {
                 getIdentityMigrationService().start();
                 updateSettingValue(MigrationContext.SOC_RDBMS_IDENTITY_MIGRATION_KEY, MigrationContext.isIdentityDone());
+
+                if (useMigrationIdentityStorage && MigrationContext.isIdentityDone()) {
+                  identityManager.setIdentityStorage(identityStorage);
```

Comment:
No, For other entities we do not have this kind of problem, there isn't any service try to create it automatically if it's not migrated.